onnxruntime/core/mlas/inc/mlas.h

```diff
@@ -709,6 +709,43 @@ MlasFindMinMaxElement(
     size_t N
     );
 
+size_t
+MLASCALL
+MlasQLinearSafePaddingElementCount(
+    size_t ElementSize,
+    size_t ElementCount
+    );
+
+void
+MLASCALL
+MlasQLinearGlobalAveragePool(
+    const uint8_t* Input,
+    float ScaleInput,
+    int32_t ZeroPointInput,
+    uint8_t* Output,
+    float ScaleOutput,
+    int32_t ZeroPointOutput,
+    size_t Channels,
+    size_t ImageSize
+    );
+
+void
+MLASCALL
+MlasNhwcQLinearGlobalAveragePool(
```

Comment:
Rename to MlasQLinearGlobalAveragePoolNhwc (same pattern as MlasReorderOutputXxx)

Reply:
Done